host-interaction/file-system/read/read-ini-file.yml

```diff
@@ -2,17 +2,21 @@ rule:
   meta:
     name: read .ini file
     namespace: host-interaction/file-system/read
-    author: "@_re_fox"
+    author:
+      - "@_re_fox"
+      - michael.hunhoff@fireeye.com
     scope: function
     examples:
       - 1d8fd13c890060464019c0f07b928b1a:0x401070
+      - E6234FB98F17201C232F4502015B47B3:0x701312FA
   features:
     - and:
       - or:
         - api: GetPrivateProfileInt
         - api: GetPrivateProfileString
+        - api: GetPrivateProfileStruct
         - api: GetPrivateProfileSection
         - api: GetPrivateProfileSectionNames
-      - string: /\.ini/i
       - optional:
+        - string: /\.ini/i
```

Comment:
making `.ini` optional to add hits for samples that don't explicitly reference the full path of the initialization file.